.env

```diff
@@ -1,2 +1,3 @@
 SECRET=sagoro
-ASECRET=bembo
\ No newline at end of file
+ASECRET=bembo
+DATABASE_URL=postgres://zfihbtyqroghuh:c14f248474d75f6ceda6a5a672f1a1eb66e5c2efaa8901bdabce620fa50b44fd@ec2-54-83-192-245.compute-1.amazonaws.com:5432/dfvmepn6rsjc7b
```

Comment:
Please remove this file from your repo.

See the links below
https://stackoverflow.com/a/52643437/4238544
https://help.github.com/en/articles/removing-sensitive-data-from-a-repository